.github/pull_request_template.md

```diff
@@ -1,8 +1,3 @@
-<!--
```

Comment:
Lol, does the header not work in templates?

Reply:
It adds the header to every PR message and that's why I removed it. 😅